style.css

```diff
@@ -410,3 +413,117 @@ button:hover {
   background: #6070ff;
   box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
 }
+
+/* Form */
+#contact-me {
+  border: 1px solid transparent;
+  background: #6070ff;
+  border-radius: 100px 0 0 0;
+  background-image: url(Assets/contact-mee.png);
+  background-position: right top;
+  background-repeat: no-repeat;
+  margin-top: 145px;
+  padding-top: 90px;
+}
+
+.form-content-container {
+  width: 100%;
+  margin: 0 24px;
+  flex-direction: column;
+}
+
+.form-header {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  width: 90%;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #fff;
+}
+
+.form-description {
+  transition: transform 2s ease-in-out;
+  width: 90%;
+  margin: 12px 0 52px 0;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #ebebff;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin-left: 24px;
+  margin-right: 24px;
+  gap: 18px;
+}
+
+.form-element,
+textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+  padding: 15px 16px;
+}
+
+textarea {
+  height: 170px;
+  width: 100%;
+  margin-top: -2px;
+  margin-bottom: 22px;
+  color: #000;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.form-element {
+  width: 100%;
+  height: 48px;
+  color: #172b4d;
+}
```

Comment:
As suggested by the previous reviewer, make sure that your design is the same as Figma's design.  You can refer the Figma design part to make it the same.
![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/101523582/4aa771b5-2735-4b3f-8995-771d91e927f4)